app/resources/views/pages/location_management/location_created.blade.php

```diff
@@ -1,9 +1,14 @@
 @extends('layouts.default')
 @section('content')
 <div class="location-created">
+	
+	@if(!empty($action))
+	<h1 class="text-center">Location Modified</h1>
+	<p class="text-center">The location has been Modified.<p>
```

Comment:
Lower case "m" for "The location has been modified." would look a little better.

Reply:
done